javascript/javascript1/week3/Distance/distance.js

```diff
@@ -0,0 +1,20 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function getTime(param1) {
```

Comment:
at the moment the appropriate name for this function would be getTimeMessage(). You can either change the name to reflect what it's doing, or better yet, return the actual time as an object (see my other comment)